dist/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>To do list</title>
+  <script defer src="to-do-list.js"></script></head>
+  <body>
+    <div id="to-do-list">
+      <ul class="to-do-list">
+        <li
+          >Today's To Do
```

Comment:
Hi John!
Well done, your work looks good, but there are still some issues that you need to work on before you move to the next project. I think it would be much better if you could put " Today's To do" in one line, for your code to look more clean and for better HTML practice, don't you think?